Resources/Private/Templates/Event/Create.html

```diff
@@ -47,10 +47,12 @@
 			<td>{event.location.location}</td>
 		</tr>
 	</f:if>
-	<tr>
-		<td><f:translate key="tx_events2_domain_model_event.video_link" /></td>
-		<td>{event.videoLink.link}</td>
-	</tr>
+	<f:if condition="{event.videoLink}">
```

Comment:
Can you please check against {event.videoLink.link}? As {event.videoLink} is only the link record where property link can be empty.